scripts/identity/assert_runtime_identity.py

```
@@ -0,0 +1,83 @@
+import argparse
+import os
+import sys
+from urllib.parse import urlparse
+
+import psycopg2
+
+
+def _require(name: str) -> str:
+    value = os.getenv(name)
+    if not value:
+        raise RuntimeError(f"{name} is required")
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)